UnityProject/Assets/Scripts/Items/Medical/Gauze.cs

```diff
@@ -31,47 +31,41 @@ public override void ServerPerformInteraction(HandApply interaction)
 			}
 		}
 
-		private void RemoveLimbExternalBleeding(LivingHealthMasterBase targetBodyPart, HandApply interaction)
+		private void RemoveLimbExternalBleeding(LivingHealthMasterBase livingHealth, HandApply interaction)
 		{
-			foreach(var container in targetBodyPart.RootBodyPartContainers)
+			foreach(var bodyPart in livingHealth.BodyPartList)
 			{
-				if(container.BodyPartType == interaction.TargetBodyPart)
+				if(bodyPart.BodyPartType == interaction.TargetBodyPart)
 				{
-					foreach(BodyPart limb in container.ContainsLimbs)
+					if(bodyPart.IsBleedingExternally)
 					{
-						if(limb.IsBleedingExternally)
+						bodyPart.StopExternalBleeding();
+						if(interaction.Performer.Player().GameObject == interaction.TargetObject.Player().GameObject)
 						{
-							limb.StopExternalBleeding();
-							if(interaction.Performer.Player().GameObject == interaction.TargetObject.Player().GameObject)
-							{
-								Chat.AddActionMsgToChat(interaction.Performer.gameObject,
-								$"You stopped your {interaction.TargetObject.Player().Script.visibleName}'s bleeding.",
-								$"{interaction.PerformerPlayerScript.visibleName} stopped their own bleeding from their {interaction.TargetObject.ExpensiveName()}.");
-							}
-							else
-							{
-								Chat.AddActionMsgToChat(interaction.Performer.gameObject,
-								$"You stopped {interaction.TargetObject.Player().Script.visibleName}'s bleeding.",
-								$"{interaction.PerformerPlayerScript.visibleName} stopped {interaction.TargetObject.Player().Script.visibleName}'s bleeding.");
-							}
+							Chat.AddActionMsgToChat(interaction.Performer.gameObject,
+							$"You stopped your {interaction.TargetObject.Player().Script.visibleName}'s bleeding.",
+							$"{interaction.PerformerPlayerScript.visibleName} stopped their own bleeding from their {interaction.TargetObject.ExpensiveName()}.");
```

Comment:
"You stopped your Clarke Hanford's bleeding." 🤔
"Clarke Hanford stopped their own bleeding from their Clarke Hanford." 😆 May as well fix this while it is in the light.
```suggestion
							Chat.AddActionMsgToChat(interaction.Performer,
							$"You stopped the bleeding on your {bodyPart}.",
							$"{interaction.PerformerPlayerScript.visibleName} stopped the bleeding on their {bodyPart}.");
```
Though `bodyPart` probably doesn't print nicely. Does it have an attributes component for `bodyPart.ExpensiveName()`?

Should probably check this before applying suggestion. Same goes for the next.

Reply:
Like I said on discord, better get the name from the BodyPartDamageDiscreptor script as it's designer defined and constant.